types/knockout/index.d.ts

```diff
@@ -4,21 +4,23 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 
-interface KnockoutSubscribableFunctions<T> {
+interface KnockoutExtensionFunctions {
     [key: string]: KnockoutBindingHandler | undefined;
+}
 
-	notifySubscribers(valueToWrite?: T, event?: string): void;
+interface KnockoutSubscribableFunctions<T> {
+    notifySubscribers(valueToWrite?: T, event?: string): void;
 }
+type KnockoutSubscribableExtensionFunctions<T> = KnockoutSubscribableFunctions<T> & KnockoutExtensionFunctions;
```

Comment:
I've wondered about whether they need to be `KnockoutBindingHandler` as well. I'm very happy to update this PR, to remove this requirement, and make the types plain interfaces. Is this the way to go?